cli/bpmetadata/cmd.go

```diff
@@ -55,6 +58,15 @@ func generate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error getting working dir: %w", err)
 	}
 
+	// validate metadata if there is an argument passed into the command
+	if len(args) == 1 {
```

Comment:
Validate seems like it should be a flag? Or maybe we should have verbs as subcommands `generate`/`validate`

Reply:
I started off with that but having a subcommand felt excessive. We still have the schema generation as a subcommand but I think want to change that to just a binary on its own since its not something that the enduser will be using. We only need it during the build process to generate the schema. Based on that, I'm OK with making this a flag.